build/scripts/create.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TEST_CLUSTER_EXISTENCE=$(gcloud container clusters list | awk "/$CLUSTER_NAME/")
+if [ -z "$TEST_CLUSTER_EXISTENCE" ]; then
+  gcloud container clusters create "$CLUSTER_NAME" --num-nodes=1 --disk-size=10
```

Comment:
I would rename this file `gke_test_create.sh` or put it in a directory like `build/scripts/tests/gke/create.sh`. The same goes for `deploy.sh`, `test.sh`, and `wait.sh`.